index.js

```diff
@@ -1,56 +1,78 @@
+      const titleBook = document.createElement('h5');
+      const authorBook = document.createElement('h5');
+      const idBook = document.createElement('h5');
+      const removeButton = document.createElement('button');
 
-  theBooks.forEach((book, i) => {
-    bookList.children[i].children[2].innerHTML = i;
-    book.id = i;
-  });
-  localStorage.setItem('booksArray', JSON.stringify(theBooks));
-}
+      bookContainer.classList.add('book-container');
+      infoContainer.classList.add('book-info');
+      titleBook.classList.add('book-title');
+      authorBook.classList.add('book-author');
+      removeButton.classList.add('remove-btn');
 
-function creaateAndAppend(title, author, id) {
-  const bookContainer = document.createElement('div');
-  const titleBook = document.createElement('h5');
-  const authorBook = document.createElement('h5');
-  const idBook = document.createElement('h5');
-  const removeButton = document.createElement('button');
-  const breakLine = document.createElement('hr');
+      titleBook.innerHTML = `${title}&nbsp;`;
+      authorBook.innerHTML = `by ${author}&nbsp;`;
+      idBook.innerHTML = id;
+      removeButton.innerHTML = 'Remove';
 
-  titleBook.innerHTML = title;
-  authorBook.innerHTML = author;
-  idBook.innerHTML = id;
-  removeButton.innerHTML = 'Remove';
+      idBook.style.display = 'none';
 
-  // idBook.style.display = 'none';
+      infoContainer.append(titleBook, authorBook, idBook);
+      bookContainer.append(infoContainer, removeButton);
+      bookList.append(bookContainer);
+
+      BookObject.remove(removeButton);
+    }
+    creaateAndAppend(this.title, this.author, this.id);
+  }
 
-  bookContainer.append(titleBook, authorBook, idBook, removeButton, breakLine);
-  bookList.append(bookContainer);
+  static remove(element) {
+    function removeBook() {
+      theBooks = theBooks.filter((book) => +book.id
+      !== +this.parentNode.children[0].children[2].innerHTML);
+      this.parentNode.remove();
 
-  removeButton.addEventListener('click', removeBook);
+      theBooks.forEach((book, i) => {
+        bookList.children[i].children[0].children[2].innerHTML = i;
+        book.id = i;
+      });
+      localStorage.setItem('booksArray', JSON.stringify(theBooks));
+    }
+
+    element.addEventListener('click', removeBook);
+  }
 }
 
 function addBooks() {
   if (
     inputTitle.value !== ''
     && inputAuthor.value !== ''
   ) {
-    creaateAndAppend(inputTitle.value, inputAuthor.value, theBooks.length);
-    const bookObj = { title: inputTitle.value, author: inputAuthor.value, id: theBooks.length };
-    theBooks.push(bookObj);
+    const newBook = new BookObject(inputTitle.value, inputAuthor.value, theBooks.length);
+    newBook.add();
+    theBooks.push(newBook);
+
     localStorage.setItem('booksArray', JSON.stringify(theBooks));
 
-    inputTitle.value = '';
-    inputAuthor.value = '';
+    // inputTitle.value = '';
+    // inputAuthor.value = '';
```

Comment:
- [ ] Kindly remove commented code. It makes your codebase look messy and hard to read.

Reply:
Most of the code reviewers mark this as optional